beacon-chain/utils/vote_cache.go

```diff
@@ -1,4 +1,4 @@
-package types
+package utils
```

Comment:
Why move vote_cache to utils package?

vote_cache is directly related to active state and used by the active state only. It won't be used anywhere else 😅

Reply:
I was facing an import cycle error so had to place it in the utils package. I didnt really want to move it haha

Reply:
so basically if we leave vote cache in the types package we have to import it into the casper package. Which will cause an import cycle error since the casper package is used extensively by methods in the types package